code/game/objects/structures/wall_frame.dm

```diff
@@ -137,6 +136,9 @@
 /obj/structure/wall_frame/hull
 	color = COLOR_HULL
 
+/obj/structure/wall_frame/prepainted
+	color = COLOR_GUNMETAL
```

Comment:
Why is this necessary? Wallframes aren't meant to be colored. They do not act like walls in the sense that they should be painted.

Also, we don't use painted versions of the wall currently.

Reply:
And that's why I made this subtype. If someone want to use prepainted walls, they will use this. If not - previous type (which is already used at the ship and colony, but has own color while walls don't).

Reply:
We won't ever use it. Ishimura won't use it, neither will Tau. Sprawl is a long shot for similar reasons. It seems you added it because you think/hope they'll be used, but I can tell you now, they won't. I've spoken to Snype about this multiple times.

Reply:
It looks a little dumb. You get mad about one new suitable type (because there already other type with same color in code) and hold the because of it PR for it. Good mappers have their own vision on their job and may see situations where current color will be better than regular blank walls.

Reply:
I'm not mad, I'm saying it's wasted effort and time.